capabilities/working-in-small-batches.md

```diff
@@ -1,52 +1,69 @@
+2. **Moderate Batches:** Batches are moderately sized, taking several weeks to complete. There may be some delays in integration and value delivery, and moderate variability, making it difficult to track progress.
+3. **Small Batches:** Work is broken down into small batches that can be completed and integrated quickly (days). There is clear visibility into progress, relatively low integration effort, and faster value delivery with some variability.
+4. **Minimal Viable Batches:** Work is decomposed into extremely small, minimal viable batches that can be completed and integrated rapidly (hours). There is clear and continuous visibility into progress, minimal integration effort, and fast value delivery with low variability.
+
+The number you selected represents your overall score for this capability. If you feel like your team or organization fits somewhere in between two scores, it's okay to use a decimal. For example, if you think your team works somewhere between moderate and small batches, then you would score a 2.5.
+
+Generally, an overall score equal to or less than 3 means you'll likely gain a lot of value from experimenting with some of the supporting practices listed here. An overall score higher than 3 generally means you and your team are largely proficient, or well on your way to becoming proficient, in the area of Working in Small Batches; you would likely benefit from evaluating your scores in other capabilities.
 
-A narrow focus on small, immediate tasks can distract from the bigger picture and long-term objectives of the project. It's important to balance the benefits of incremental work with broader strategic planning to ensure that all updates represent meaningful contributions to the project's overarching goals.
 
 ## Supporting Practices
+The following is a curated list of supporting practices to consider when looking to improve your team's Working in Small Batches capability. While not every practice will be beneficial in every situation, this list is meant to provide teams with fresh, pragmatic, and actionable ideas to support this capability.
+
 
 ### Target an MVP
 
-Start with defining a clear Minimum Viable Product (MVP)—the simplest version of a product or feature that delivers value to users and allows the team to gather actionable feedback quickly. Focus on identifying core user needs, then delivering just those. A good MVP should be achievable quickly, typically within weeks rather than months, to ensure rapid feedback.
+Start with defining a clear Minimum Viable Product (MVP). This is the simplest version of a product or feature that delivers value to users and allows the team to gather actionable feedback quickly. Focus on identifying core user needs, then work in small batches to deliver on just those. A good MVP should be achievable quickly, typically within weeks rather than months, to ensure rapid feedback.
```

Comment:
I know this is me commenting on my own words, but what if we frame the follow-on work as building small layers on top of the MVP? Our usage of `just those` struck me as a little myopic.